CONTRIBUTING.md

```diff
@@ -1,39 +1,164 @@
+## Common issues and resolutions
+
+- Using `grape-active_model_serializers`, or any non-Rails server. See
+    [issue](https://github.com/rails-api/active_model_serializers/issues/1258).
+-
 ## How can I help?
 
-Everyone is encouraged to open issues that are affecting you: bugs, ideas, performance problems – everything helps!
+### Filing an issue
+
+Everyone is encouraged to open issues that are affecting you:
+bugs, ideas, documentation, performance problems – everything helps!
+
+#### Before
+
+1. The first place to start is by looking at our [GitHub Issues](https://github.com/rails-api/active_model_serializers/issues).
+
+  - Check if your issue has already reported.
+  - If you find an existing issue report, feel free to add further information to that report.
+
+
+#### Writing
+
+If possible, please include the following information when [reporting an
+issue](https://github.com/rails-api/active_model_serializers/issues/new):
+
+- ActiveModelSerializers version.
```

Comment:
ActiveModelSerializers version (0.8, 0.9, 0.10, master)

Maybe drop the dot at the end.